android/templates/module/generated/build.gradle

```diff
@@ -247,10 +247,14 @@ publishing {
 				list.addAll(configurations.implementation.allDependencies)
 				list.each {
 					if ((it.group != null) && (it.name != null)) {
-						def childNode = dependenciesNode.appendNode('dependency')
-						childNode.appendNode('groupId', it.group)
-						childNode.appendNode('artifactId', it.name)
-						childNode.appendNode('version', it.version)
+
+						// Skip Titanium SDK dependency.
+						if (it.group != 'org.appcelerator') {
+							def childNode = dependenciesNode.appendNode('dependency')
+							childNode.appendNode('groupId', it.group)
+							childNode.appendNode('artifactId', it.name)
+							childNode.appendNode('version', it.version)
+						}
```

Comment:
Nice catch. I don't think we should remove our Titanium library as a dependency though. We should be able to set the version to the minsdk like this.
```groovy
if ((it.group != null) && (it.name != null)) {
	def childNode = dependenciesNode.appendNode('dependency')
	childNode.appendNode('groupId', it.group)
	childNode.appendNode('artifactId', it.name)
	if ((it.group == 'org.appcelerator') && (it.name == 'titanium')) {
		childNode.appendNode('version', <%- minSdkVersion %>)
	} else {
		childNode.appendNode('version', it.version)
	}
}
```

I coded the above without testing it. So, you'll need to double check it.

Reply:
Hold on... my above code is wrong. Don't use `minSdkVersion` because that's the min API Level.

In our "_buildModule.js", we need to pass `manifest.minsdk` as `tiMinSdkVersion` and use that instead.